packages/zero-cache/src/services/view-syncer/view-syncer.ts

```diff
@@ -1079,6 +1067,7 @@ export class ViewSyncerService implements ViewSyncer, ActivityBasedService {
           span.setAttribute('table', transformedAst.table);
           for (const _ of this.#pipelines.addQuery(
             transformationHash,
+            hash,
```

Comment:
@darkgnotic I think this is correct but I wanted to verify with you. I see in some other places we map from the transformationHash to multiple client side hashes (aka queryID).

What happens if two queries ends up with the same transformationHash? Do they share the IVM pipeline? If so, then I would need to map the ID similar to how we do it in other places.